eng/Version.Details.xml

```diff
@@ -29,17 +29,17 @@
       <Uri>https://github.com/microsoft/usvc-apiserver</Uri>
       <Sha>cc584ed90a91ef8b818f9537ffd1d11639c2448a</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Extensions.Http.Resilience" Version="8.9.1">
-      <Uri>https://github.com/dotnet/extensions</Uri>
-      <Sha>544b8d5c8de9ed4940c5f39457256d090b8ffbfb</Sha>
+    <Dependency Name="Microsoft.Extensions.Http.Resilience" Version="8.10.0">
```

Comment:
This is out of date / incorrect. We are using the 9.0 versions from dotnet/extensions now.

Reply:
Typically we haven't been flowing nightly builds of dotnet/extensions packages (so that `main` stays on a ship-ready state), and we don't have a public build of dotnet/extensions yet which includes the TFM dependency changes, so my plan was to keep this as-is here, and then once we ship 9.0, the merge from internal -> main would contain those fixes.

All that said, I can still change this if you want me to and flow a nightly build.